inc/sailag.h

```diff
@@ -167,6 +167,15 @@ typedef enum _sai_lag_attr_t
      */
     SAI_LAG_ATTR_SYSTEM_PORT_AGGREGATE_ID,
 
+    /**
+     * @brief Anchor attribute used to unique identify empty LAG.
+     *
+     * @type sai_uint64_t
+     * @flags CREATE_ONLY
+     * @default 0
+     */
+    SAI_LAG_ATTR_ANCHOR,
```

Comment:
Can we make it more meaningful (string instead of u64)?

The reason for that is that SAI can be able to pass this metadata to vendor SDK, which will be displayed in the debug dump and other tools. Otherwise, SAI will just ignore it anyway.

Reply:
i actually wanted to make this simple as possible, previous approach #1080 was adding string here, i wanted this be simpler to store for sonic and vendors

Reply:
addressed meeting notes, a changed to create_and_set for warm boot scenario, and also changed to char type which corresponds to chardata in sai_attribute_value_t which can be up 32 bytes. This we would also assume that this will be some ascii user readable string